types/node/stream.d.ts

```diff
@@ -34,6 +34,7 @@ declare module 'stream' {
     import Readable = internal.Readable;
     import ReadableOptions = internal.ReadableOptions;
     class ReadableBase extends Stream implements NodeJS.ReadableStream {
+        compose: Function;
```

Comment:
This is missing the correct typings for `compose`, see https://nodejs.org/api/stream.html#readablecomposestream-options for more details.

Reply:
thank you, I'll have a look